### PR TITLE
Feature/connect wallet modal v2

### PR DESCRIPTION
**Pull request by hachiojidev**
Redesign wallet modal. Note - further functionality will be in a separate PR.

image
image

### CODE REVIEW

File: packages/pancake-uikit/src/components/Svg/index.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ export { default as ArrowForwardIcon } from "./Icons/ArrowForward";
 export { default as ArrowUpIcon } from "./Icons/ArrowUp";
 export { default as AutoRenewIcon } from "./Icons/AutoRenew";
 export { default as BinanceIcon } from "./Icons/Binance";
+export { default as BinanceChainIcon } from "./Icons/BinanceChain";
```
Comment (hachiojidev) [inline packages/pancake-uikit/src/components/Svg/index.tsx:10]: Moved wallet icons here

File: packages/pancake-uikit/src/widgets/WalletModal/config.tsx [hunk 2]
```diff
@@ -1,53 +1,63 @@
...
+import Coin98 from "../../components/Svg/Icons/Coin98";
+
 import { Config, ConnectorNames } from "./types";
 
 const connectors: Config[] = [
   {
     title: "Metamask",
     icon: Metamask,
     connectorId: ConnectorNames.Injected,
+    priority: 1,
```
Comment (hachiojidev) [inline packages/pancake-uikit/src/widgets/WalletModal/config.tsx:17]: This enables use to display different wallets on the initial view.

File: packages/pancake-uikit/src/components/Svg/index.tsx [hunk 3]
```diff
@@ -40,13 +42,16 @@ export { default as LaurelRightIcon } from "./Icons/LaurelRight";
...
 export { default as LogoRoundIcon } from "./Icons/LogoRound";
+export { default as MathWalletIcon } from "./Icons/MathWallet";
 export { default as MedalBronzeIcon } from "./Icons/MedalBronze";
 export { default as MedalGoldIcon } from "./Icons/MedalGold";
 export { default as MedalPurpleIcon } from "./Icons/MedalPurple";
 export { default as MedalSilverIcon } from "./Icons/MedalSilver";
 export { default as MedalTealIcon } from "./Icons/MedalTeal";
 export { default as MetamaskIcon } from "./Icons/Metamask";
 export { default as MinusIcon } from "./Icons/Minus";
+export { default as MoreHorizIcon } from "./Icons/MoreHoriz";
```
Comment (RabbitDoge) [inline packages/pancake-uikit/src/components/Svg/index.tsx:53]: `MoreHorizontal` and `MoreVertical`

File: packages/pancake-uikit/src/widgets/WalletModal/ConnectModal.tsx [hunk 4]
```diff
@@ -1,40 +1,95 @@
...
-    </HelpLink>
-  </Modal>
-);
+/**
+ * Checks local storage if we have saved the last wallet the user connected with
+ * If we find something we put it at the top of the list
+ *
+ * @returns sorted config
+ */
+const getPreferredConfig = (walletConfig: Config[]) => {
```
Comment (RabbitDoge) [inline packages/pancake-uikit/src/widgets/WalletModal/ConnectModal.tsx:30]: this logic should be in the app now, inside the user reducer (like all the config related to the user). The reducer is already serialized so we don't have to manipulate the local storage directly

File: packages/pancake-uikit/src/widgets/WalletModal/ConnectModal.tsx [hunk 5]
```diff
@@ -1,40 +1,95 @@
...
+        <Box p="24px">
+          <Text textAlign="center" color="textSubtle" as="p" mb="16px">
+            Haven&#39;t got a crypto wallet yet?
+          </Text>
+          <Button
+            as="a"
+            href="https://docs.pancakeswap.finance/get-started/connection-guide"
+            variant="subtle"
+            width="100%"
+            target="_blank"
```
Comment (RabbitDoge) [inline packages/pancake-uikit/src/widgets/WalletModal/ConnectModal.tsx:84]: you can use `as={Link}` and the `external` props

**Review by RabbitDoge — APPROVED** [review]

**Comment by RabbitDoge** [comment]
Formating issue

File: packages/pancake-uikit/src/widgets/WalletModal/ConnectModal.tsx (hunk 5 above)
Comment (hachiojidev) [reply packages/pancake-uikit/src/widgets/WalletModal/ConnectModal.tsx:84]: I tried that initially but it messages with the styling. You cannot seem to apply other styles like `width="100%"` and `display="block"`.

I did however, use the `getExternalProps` helper.

**Pull request merged**